backend/dal/Services/IPropertyService.cs

```diff
@@ -0,0 +1,15 @@
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Entities.Views;
+
+namespace Pims.Dal.Services
+{
+    /// <summary>
+    /// IPropertyService interface, provides functions to interact with properties within the datasource.
+    /// </summary>
+    public interface IPropertyService
+    {
+        Paged<Property> Get(AllPropertyFilter filter);
+        Paged<Property> Get(ParcelFilter filter);
+        Paged<Property> Get(BuildingFilter filter);
```

Comment:
I think I mentioned before but what would you think of naming these `GetPage` to follow the convention from the other services?